app/_src/deck/guides/konnect.md

```diff
@@ -206,8 +206,12 @@ The default API decK uses is `https://us.api.konghq.com`, which targets the `clo
 
 {% if_version gte:1.14.x %}
 
-{{site.base_gateway}} supports US and EU geographic regions.
-To target the EU region, set `konnect-addr` to `"https://eu.api.konghq.com"`.
+{{site.base_gateway}} supports US, EU, and AU [geographic regions](/konnect/geo/).
+
+To target a specific geo, set `konnect-addr` to one of the following:
```

Comment:
@cloudjumpercat My suggestion, and please take it as such, is that we should alphabetize lists of geos so it doesn't appear we have unconscious US bias. WDYT?

Reply:
Sure, I'll update those!